apps/storybook/src/FileUpload.stories.tsx

```diff
@@ -74,3 +82,73 @@ export const WrappingInput: Story<FileUploadProps> = (args) => {
 WrappingInput.args = {
   dragContent: 'Drop file to upload',
 };
+
+export const SingleFileUpload: Story<FileUploadProps> = (args) => {
+  return (
+    <FileUpload
+      {...args}
+      onFileDropped={(files) => {
+        action(`${files.length} files uploaded`)();
+      }}
+    >
+      <FileUploadCard />
+    </FileUpload>
+  );
+};
+
+export const SingleFileUploadCustom: Story<FileUploadProps> = (args) => {
+  const inputRef = React.useRef<HTMLInputElement>(null);
+  const [files, setFiles] = React.useState<File[]>([]);
+  const fileInputId = React.useId();
+
+  const emptyCard = (
+    <FileEmptyCard>
+      <FileEmptyCard.Icon>
+        <SvgSmileySadVery />
+      </FileEmptyCard.Icon>
+      <FileEmptyCard.Text>
+        <FileEmptyCard.Anchor label='Custom Label Text'>
+          <FileUploadCard.Input name={fileInputId} ref={inputRef} />
+        </FileEmptyCard.Anchor>
+        <FileEmptyCard.Description>
+          Custom Description Text
+        </FileEmptyCard.Description>
+      </FileEmptyCard.Text>
+    </FileEmptyCard>
+  );
+  return (
+    <FileUpload
+      {...args}
+      onFileDropped={(files) => {
+        action(`${files.length} files uploaded`)();
+      }}
+    >
+      <FileUploadCard
```

Comment:
`FileUploadCard.Anchor -> FileUploadCard.InputLabel`
	
`FileUploadCard.Label -> FileUploadCard.Title`

`FileUploadCard.Text -> FileUploadCard.Info` to show that information about the file should be shown here. Not necessarily the best name since users can add custom text to `Title` and `Description` that has nothing to do with the file.
	
I'm having trouble understanding why `FileEmptyCard` should be a subcomponent of `FileUploadCard` if they are two different states?

Reply:
> I'm having trouble understanding why FileEmptyCard should be a subcomponent of FileUploadCard if they are two different states?

because `FileEmptyCard` is used inside `FileUploadCard`. they are not alternatives to each other. `FileUploadCard` is the "main" component which holds all the logic. but it's upto you, i can see both options working fine.

---

also this one is still unaddressed:

> FileUploadCard feels wrong because cards are usually elevated and have a background color. Maybe we should add those things?

if not elevation, then a border would also do.

Reply:
Elevation and background color have been added